src/main/java/dev/jlibra/admissioncontrol/query/UpdateToLatestLedgerResult.java

```diff
@@ -0,0 +1,27 @@
+package dev.jlibra.admissioncontrol.query;
+
+import java.util.List;
+
+import dev.jlibra.AccountState;
+
+public class UpdateToLatestLedgerResult {
+
+    private List<AccountState> accountStates;
+
+    private UpdateToLatestLedgerResult() {
+    }
+
+    public static UpdateToLatestLedgerResult create() {
```

Comment:
Can we have public constructor and remove `create()`?

Reply:
Yes, currently the value objects are a mess, and there are several different ways of making them (public constructors, builder methods etc). I think we should come up with a solution for all of them, I'll create an issue about this one too. We can then discuss there how to do these properly.